backend/app/literature/schemas/reference_schemas.py

```diff
@@ -52,6 +52,18 @@ class Config():
         orm_mode = True
         extra = "forbid"
 
+    @validator('title')
```

Comment:
This is needing to be cleaned up a little. The fact that the title and category are not listed as optional means that it will never be None. I guess what you are adding is that the title also can't be a blank string. The category doesn't need the validator because it has to be populated with one of the options from the enum. In general this type of validation is only required for the PATCH enpoints... as the required fields in the POST should not be None if they are supplied in the PATCH. Hopefully that makes sense.